### PR TITLE
Minor improvements, mostly coding style

### PR DESCRIPTION
**Pull request by linusg**
Hey @ritiek,

nice project! For now I've just looked around the code but I'll try it out for sure tomorrow. Just proposing a few changes, see my attached explanations.

Cheers!

### CODE REVIEW

File: muxnect/server.py [hunk 1]
```diff
@@ -98,7 +97,7 @@ def handle_request(window_name):
         except LibTmuxException:
             pass
 
-    return '200'
+    return '', 200
```
Comment (linusg) [inline muxnect/server.py:100]: Seems like you want to respond with a simple response body but a status code of 200. For sure you could technically send any string as the response and flask will make a 200 out of it, but IMHO (and let me know what you think) the most clean way, according to the Zen of Python, is explicitly returning a tuple containing the empty response and status code.

File: muxnect/server.py [hunk 2]
```diff
@@ -144,13 +142,12 @@ def command_line():
...
-    print('Listening on {}'.format(url))
+    print('Listening on {0}'.format(url))
     print('Press CTRL+C to exit muxnect')
 
     if not detach:
         session.attach_session()
 
 
 if __name__ == '__main__':
-
```
Comment (linusg) [inline muxnect/server.py:155]: Seems to be a completely unnecessary empty line, along with 1 or 2 others :thinking:

File: muxnect/server.py [hunk 3]
```diff
@@ -110,9 +109,9 @@ def command_line():
     bind_address = args.bind_address
     port = args.port
 
-    socket_code =  port_is_busy(port)
+    socket_code = port_is_busy(port)
     if socket_code:
-        raise OSError('[Errno {}] Cannot start web server on port {}'.format(socket_code, port))
+        raise OSError('[Errno {0}] Cannot start web server on port {}'.format(socket_code, port))
```
Comment (linusg) [inline muxnect/server.py]: Just for consistency, it works without the index but you use them at some other ``print``s, even though it's more verbose it's explicit.

File: muxnect/server.py (hunk 2 above)
Comment (ritiek) [reply muxnect/server.py:155]: I've always wondered whether it looks good to have a blank line after `__name__ == '__main__'` but after some research, seems like I need to fix myself. Thanks!

File: muxnect/server.py (hunk 3 above)
Comment (ritiek) [reply muxnect/server.py]: Yep, good idea. Although, we should set index for port to `{1}` as well.

File: muxnect/server.py (hunk 2 above)
Comment (linusg) [reply muxnect/server.py:155]: About the second index missing, I've simply overseen that and fixed it though. Thanks.
Regarding the blank line in the "`ifmain`" block, seems to be a question of general preference; even though I don't put one there myself and it does not seem very common, as you've noticed too. I'm curious what PEP8 would say about this one :smile:

**Pull request merged**